docs/docs/services/qix-engine/access-control.md

```diff
@@ -114,20 +128,22 @@ The comparison operators are:
 | [`like`](#like) | Wildcard string matching |
 | [`matches`](#matches) | Regular expression string matching |
 
-### The `user` Object
+### The `user` object
 
-All rules are evaluated in the context of a _User_. In rule expressions the user is represented by the `user` object.
+All rules are evaluated in the context of a user.
+In rule expressions, the user is represented by the `user` object.
 
-#### Attributes
+#### User attribute
 
 | Attribute | Description |
 | --------- | ----------- |
-| `user.id` | The identifier of the user. This gets the value of the mandatory `sub` attribute in the JWT. |
+| `user.id` | The identifier of the user. This `user.id` gets the value of the mandatory `sub` attribute in the JWT. |
 
-In addition to this `user` will contain all other attributes defined in the
-[JWT header used to authenticate the user](../../tutorials/authorization.md#json-web-token).
+Also, `user` contains all attributes defined in the JWT header that are used to authenticate the user.
 
-**Examples**
+For more information about attributes in the JWT header, see [JSON web token](../../tutorials/authorization.md#json-web-token).
+
+##### Example of user object
```

Comment:
Avoid making sections here since the TOC becomes very cluttered. That's why the formatting is **bold**.

Reply:
I agree but I changed this because I thought linting would fail when emphasis used to mark headings.
TOC is major drawback to building a helpsite with Mkdocs... :)